cypress/e2e/dapps.cy.ts

```diff
@@ -0,0 +1,19 @@
+/// <reference types="cypress" />
+
+import data from '../../src/valora-dapp-list.json'
+
+const ignoreList = [
+  'gooddollar', // Blocks Microsoft services via Cloudflare
+]
+
+describe('Dapp Up Check', () => {
+  for (const application of data.applications) {
+    if (application.url.startsWith('celo://')) continue
+    if (ignoreList.includes(application.id)) continue
+    it(`should return 200 for ${application.url}`, () => {
```

Comment:
use `it.each`?

Reply:
Cypress doesn't ship with `it.each` out of the box unfortunately. There is a relatively popular plugin for it if we want this functionality: [cypress-each](https://www.npmjs.com/package/cypress-each).